src/lib/lforms/lhc-form-validation.js

```diff
@@ -127,7 +127,7 @@ const Validation = {
           valid = regex.test(value);
           break;
         case "TM":  // time
-          var regex = /^\s*(((\d|[0-1]\d|2[0-4]):([0-5]\d))|(\d|0\d|1[0-2]):([0-5]\d)\s*([aApP][mM]))\s*$/;
+          var regex = /^([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?$/;
```

Comment:
- It looks like this change removes support for 12 hour time format (8:00AM). Do we ever use that?
- Why is 60 allowed in the seconds place? Is `12:15:60` really a valid time? Wouldn't that be written as `12:16:00` instead?

Reply:
- We are supporting the 24 hrs time format for new sapphire.
- I will update the regex for 2nd point

Reply:
after looking into this more, I think the 60 is for leap second. We should add the 60 back in.

Reply:
Ok, I have updated the regex